mithril.js

```diff
@@ -28,6 +28,19 @@
 		return "v0.2.5"
 	}
 
+	var div = document.createElement("div")
+	var frag = document.createDocumentFragment()
+
+	function toDOM(HTML){
+		div.innerHTML = HTML
```

Comment:
A fragment is a bit abstract in this sense: it is never appended to another node directly (it is impossible for a node to contain a fragment), rather it's children are appended and the fragment is emptied.